core/embed/prodtest/main.c

```diff
@@ -364,6 +393,124 @@ static void test_touch(const char *args) {
   touch_deinit();
 }
 
+static void test_touch_custom(const char *args) {
+#define NUM_PARAMS 5
+
+  int params[NUM_PARAMS] = {0};
+  int num_params = 0;
+
+  extract_params(args, params, &num_params, NUM_PARAMS);
```

Comment:
Maybe it would be easier and more readable to use `sscanf` here instead of own `extract_params`.

Reply:
i agree that could be an improvement from readability and maintainability POV, but it doesn't compile with our current std lib due to `undefined reference to _sbrk` and don't want to try figuring out what needs to be changed and implications of it right now